gui/slick/interfaces/default/inc_bottom.mako

```diff
@@ -66,11 +66,15 @@
         <!--
```

Comment:
If no one can find any reason for it, that's more code we can remove. 

@WebSpider @neoatomic @fernandog @miigotu